src/collections/_documentation/platforms/python/django.md

```diff
@@ -66,3 +66,32 @@ The following keyword arguments can be passed to `DjangoIntegration()`:
 
 The user feedback feature can be used with this integration.  For more information
 see [User Feedback]({% link _documentation/enriching-error-data/user-feedback.md %}?platform=django).
+
+## Reporting other status codes
+
+In some situations it might make sense to report 404s and other errors besides 500 to Sentry. This can be achieved by writing your own Django view for those status codes. For example:
```

Comment:
> other errors besides 500

Are 500 errors literally all we capture, or by default do we capture all 500-level errors (501, 502, etc)?

Reply:
Only 500. We don't have some code like `if status_code == 500` in there, but rely on Django to tell us what is an unhandled exception.

Reply:
your call on this one, no opinion

Reply:
Yeah, I like "besides general unhandled exceptions" better - it's closer to the full story without getting too deep into the weeds.